### ISSUE TITLE
Not compatible with python3

### ISSUE THREAD

**Issue by siznax** [siznax/wptools#2]
Open to pull-requests for this... :octocat:

**Comment by siznax** [comment]
Progress on this is happening in #13

**Comment by siznax** [comment]
Working through python3 modernization described here: https://docs.python.org/3/howto/pyporting.html

**Comment by siznax** [comment]
After some pylint fixes c1192ce66f8352041277987613d4fdb664be9d02 the relevant blockers seem to be:

```
************* Module wptools.core
E: 14, 0: Unable to import 'urlparse' (import-error)
E:567,20: Module 'urllib' has no 'quote' member (no-member)
E:569,20: Module 'urllib' has no 'quote' member (no-member)
************* Module wptools.utils
E: 34,15: Module 'urllib' has no 'quote' member (no-member)
```

**Comment by siznax** [comment]
The python3 fixes d979184f3135e353d1a4e93507b3a0be255c328d  needed were basically:

**wptools.core**:

```python
try:  # python2
    from urllib import quote
    from urlparse import urlparse
except ImportError:  # python3
    from urllib.parse import quote, urlparse


def is_text(obj):
    """
    returns True if object is text-like
    """
    try:  # python2
        return isinstance(obj, basestring)
    except NameError:  # python3
        return isinstance(obj, str)


def json_loads(data):
    """
    python-version safe json.loads
    """
    try:  # python2
        return json.loads(data)
    except TypeError:  # python3
        return json.loads(data.decode('utf-8'))
```

**wptools.utils**:

```python
try:  # python2
    from urllib import quote
except ImportError:  # python3
    from urllib.parse import quote

def snip_html(text, verbose=0):
    ...
    keep_html = lxml.html.tostring(keep, encoding='unicode')
```

Not too bad!
